schema_cache.rst

```diff
@@ -106,26 +106,50 @@ To refresh the cache without restarting the PostgREST server, send the server pr
      # or in docker-compose
      docker-compose kill -s SIGUSR1 <service>
 
-The above is the manual way to do it. To automate cache reloads, use a database trigger like this:
+Another option is to send a `database notification event <https://www.postgresql.org/docs/current/sql-notify.html>`_ using the ``NOTIFY`` command. To enable this feature, you need to add these variables to the configuration file:
+
+.. code::
+
+  # postgrest.conf
+
+  # The name of the notification channel PostgREST will listen to
+  db-channel = pgrst
```

Comment:
`db-channel=pgrst` is set by default, I think we can omit this setting. `db-channel-enabled=true` is the only requisite.
```suggestion
```